### DISCUSSION TITLE
How to add ".well-known" folder into my hexo server?

### DISCUSSION THREAD

**Discussion by lawbyte** [hexojs/hexo#5567] [Question / Answer]
i want to add that folder so when i `hexo generate` the .well-known and the files inside it is generated into the public folder
im tryin to use skip-render or add it in my themes/name-themes/sources/.well-known its still dosnt work, please help me

**Comment by lawbyte** [comment]
hello?

**Comment by uiolee** [comment]
https://hexo.io/docs/configuration#Include-Exclude-Files-or-Folders
